soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -58,14 +64,14 @@
 	#endif
 #elif CONFIG_SOC_RISCV_TELINK_B95
 	#if DT_ENUM_IDX(DT_NODELABEL(power), power_mode) == 0
-		#define POWER_MODE      LDO_1P4_LDO_2P0
+		#define POWER_MODE      LDO_0P94_LDO_1P8
 	#elif DT_ENUM_IDX(DT_NODELABEL(power), power_mode) == 1
-		#define POWER_MODE      DCDC_1P4_LDO_2P0
+		#define POWER_MODE      DCDC_0P94_LDO_1P8
 	#elif DT_ENUM_IDX(DT_NODELABEL(power), power_mode) == 2
-		#define POWER_MODE      DCDC_1P4_DCDC_2P0
+		#define POWER_MODE      DCDC_0P94_DCDC_1P8
 	#else
-		#error "Wrong value for power-mode parameter"
-	#endif
+	#error "Wrong value for power-mode parameter"
+#endif
```

Comment:
check tabs, for this #endif it is missed

Reply:
fixed